app/frontend/styles/base/_utilities.scss

```diff
@@ -73,6 +73,14 @@ b {
   }
 }
 
+.two-column-list {
+
+  column-count: 2;
+  @include govuk-media-query($until: tablet) {
```

Comment:
you dont need this media query as column-count is already 2?

Reply:
Make sense!